src/wallet/wallet2.cpp

```diff
@@ -7454,6 +7461,12 @@ std::vector<wallet2::pending_tx> wallet2::create_transactions_2(std::vector<cryp
   for (size_t i = 0; i < m_transfers.size(); ++i)
   {
     const transfer_details& td = m_transfers[i];
+    const uint64_t fractional_threshold = fee_multiplier * fee_per_kb/1024 * estimate_tx_size(use_rct, 1, fake_outs_count, -1, 0, bulletproof) / 10;
```

Comment:
You're right, thanks.

The 10 is an arbitrary safety margin to place the threshold at 10% of the estimated size in case the estimate is an overestimation. I think it can be dropped. What do you think?